src/utility/filter-functions.js

```diff
@@ -0,0 +1,56 @@
+const getSortByPrice = (productList, sortBy) => {
+  const sortByPriceList = [...productList];
+  sortBy
+    ? sortByPriceList.sort(
+        (a, b) => Number(b.discountPrice) - Number(a.discountPrice)
+      )
+    : sortByPriceList.sort(
+        (a, b) => Number(a.discountPrice) - Number(b.discountPrice)
+      );
+  return sortByPriceList;
+};
+
+const getSortByCategory = (productList, categoryName) => {
+  const sortByCategoryList = [...productList];
+  const { yogamats, pants, shirts } = categoryName;
+
+  if (yogamats && pants && shirts) return sortByCategoryList;
+  if (yogamats && pants)
```

Comment:
you can use switch here as it would be helpful in case you add more categories later.

Reply:
I will do it later.